src/content/developers/docs/consensus-mechanisms/pos/faqs/index.md

```diff
@@ -0,0 +1,173 @@
+
+- Read more about [defending Ethereum proof-of-stake from attackers](src/content/developers/docs/consensus-mechanisms/pos/attack-and-defense)
+- Read more about [proof-of-stake design](https://medium.com/@VitalikButerin/a-proof-of-stake-design-philosophy-506585978d51)
+
+## Does proof-of-stake make Ethereum cheaper? {#does-pos-make-ethereum-cheaper}
+
+No. The cost to send a transaction is determined by a dynamic fee market that increases when there is more network demand. This is not directly influenced by the consensus mechanism.
+
+Read more about [gas](src/content/developers/docs/gas).
+
+## What are nodes, clients and validators? {#what-are-nodes-clients-and-validators}
+
+Nodes are computers connected to the Ethereum network. Clients are the software they run that turns the computer into a node. There are two types of client: execution clients are consensus clients. Both are needed in order to create a node. A validator is an optional add-on to a consensus client that enables the node to participate in proof-of-stake consensus. This means creating and proposing blocks when they are selected, and attest to blocks they hear about on the network. In oorder to run a validator, the node operator must deposit 32 ETH into the deposit contract.
+
+- Read more on [nodes and clients](src/content/developers/docs/nodes-and-clients)
+- Read more about [staking](src/content/staking)
+
+## Is proof-of-stake a new idea? {#is-pos-new}
+
+No. The basic idea of proof-of-stake was proposed on BitcoinTalk forum as an upgrade to Bitcoin. It was eleven years before it was ready to implement on Ethereum Mainnet. Some other chains implemented proof-of-stake earlier than Ethereum, but not Ethereum's specific mechanism (known as Gasper).
+
+- Read the [BitcoinTalk blogpost](https://bitcointalk.org/index.php?topic=27787.0)
+- Read more on [Gasper](src/content/developers/docs/consensus-mechanisms/pos/gasper)
```

Comment:
Actually, we don't need this link here IMO. It's more relevant to the question below.

```suggestion
```